### ISSUE TITLE
Electoral district validation

### ISSUE THREAD

**Issue by deivid-rodriguez** [podemos-info/decidim-module-votings#37]
A user should be given an error if:

* A voting adds two different "voting identifiers" configured for the same scope.

There's other case that comes to my mind, but I'd say it's actually valid? Namely, if a voting configures the same "voting identifier" for two different scopes.

**Comment by leio10** [comment]
Right, two electoral districts for the same scope is invalid, regardless of the voting identifier value, and two electoral districts with the same voting identifier and different scopes are perfectly valid.
